maintenance/scripts/lib/load_nvm.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env sh
+
+# In the hashbang line above, we use `sh` instead of `bash` as the script
+# shell, since `sh` is more portable across systems. Be aware of differences
+# with built-in functions and flow control syntax: e.g., `.` vs `source`
+# notation, `[ ]` vs `[[ ]]` conditionals, etc.
+# See: https://stackoverflow.com/a/5725402
```

Comment:
I'd like to later move this up to top-level, and convert all scripts to `sh`, but out-of-scope here. Will do it in a more suitable PR.

Reply:
I like it.  Throw it up on the board and let's do it!

Reply:
I'd like to move all the update_* scripts to the lib folder, too.